frontend/test/lit/test_custom_ops.py

```diff
@@ -41,6 +41,7 @@ def decomposition(self):
 class CustomDeviceWithoutSupport(qml.QubitDevice):
     name = "Device without support for RXX gate."
     short_name = "dummy.device"
+    backend = "default"
```

Comment:
It probably works for the test, but given our recent discussion on this parameter being an instance attribute rather than a class attribute we should move this line to the init function to be consistent.

Reply:
Good catch! :smile: Fixed them all.